src/api.cpp

```diff
@@ -0,0 +1,63 @@
+//
+// Created by clemens on 24.06.25.
+//
+
+#include "api.hpp"
+// clang-format off
+#include "ch.h"
+#include "hal.h"
+// clang-format on
```

Comment:
What would it be without the exception? I think it should be `<ch.h>` and `<hal.h>` because they will never be found locally.